docs/reference/query-dsl/constant-score-query.asciidoc

```diff
@@ -7,12 +7,16 @@ filter. Maps to Lucene `ConstantScoreQuery`.
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
+  "query": {
```

Comment:
Good point - this really was more me being too lazy to reformat than anything else, but I agree it makes a lot of sense to keep these the same within one file.